designs/configurable-registeration-ttl.md

```diff
@@ -0,0 +1,60 @@
+# RFC: Configurable registeration TTL
+
+## Motivation
+
+[Issue](https://github.com/kubernetes-sigs/karpenter/issues/357)
+
+Karpenter currently hardcodes [registeration TTL](https://github.com/kubernetes-sigs/karpenter/blob/main/pkg/controllers/nodeclaim/lifecycle/liveness.go#L44) which means nodes must take < 15 minutes to come up and register with the Kubernetes control plane.
```

Comment:
Do we know what instances are taking that long? Do you have a list of instance types that frequently hit this TTL?  We ran into some issues before but improved our latency so we don't hit that anymore.

Reply:
I don't have a specific set of instance types unfortunately